test/docker/Test.dockerfile

```diff
@@ -6,7 +6,7 @@
 # a script to configure kubectl, potentially install Helm, and run the tests
 # manually. This image only has the dependencies pre-installed.
 
-FROM alpine:3.11
+FROM circleci/golang:1.14
```

Comment:
I haven't built an image yet, but this change is needed so that we can run these tests in CI.